### DISCUSSION TITLE
长草了，求除草配方，在线等急

### DISCUSSION THREAD

**Discussion by LT-UT** [share121/inter-knot#2793] [其他]
50级后就没撒事做了，就冰队这个在练。这版本就战斗，爽，但没多少丁尼宝箱了。那个预言之下多好，长，但剧情好，丁尼也多。我现在穷的只剩菲灵了，我是雅****,先不想抽，等。绝区啥时才能加联机打怪啊。草长起来了，求除草配方。
![R-C](https://github.com/user-attachments/assets/25c75da0-4f8c-4df4-bede-c3cdf6229c37)

**Comment by HKLHaoBin** [comment]
米哈游游戏的通病 只能等版本更新

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/8b9838175cb6312df34c32648a8e6649_1050729359995605746.png" alt="景元 吸猫" width="200" >

**Reply by LT-UT** [reply]
等，可我草多快要长满房间里了。还被确诊电子阳痿了，现在就只有绝区零才能缓解。有pc游戏推荐吗，让我除草。

**Reply by ysd1911** [reply]
等二十号玩黑猴<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Reply by LT-UT** [reply]
配置不行，官方测试工具测试了不行

**Reply by ysd1911** [reply]
啊？哪里有官方测试工具，我去试试，是steam上就有吗？

**Reply by LT-UT** [reply]
yes

**Comment by isle200inf** [comment]
玩自动化吗<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by LT-UT** [reply]
英文名叫啥
我了解一下

**Reply by isle200inf** [reply]
异形工厂2，刚出的，一个造产线的游戏

**Reply by LT-UT** [reply]
过一会玩玩

**Reply by Merce-Odyssey** [reply]
如果你能接受PVE一关1个小时的话，有款游戏叫Mindustry，github上面就有

**Reply by LT-UT** [reply]
Merce-Odyssey
sorry,这个游戏不是我的菜

**Comment by Lanxey** [comment]
我今天才47，刚做完卧底蓝调，还有七八个支线和代理人任务没做，不急

**Reply by LT-UT** [reply]
额，等你做完就长了，除非你能把这些任务到下个版本更新刚好做完。

**Comment by lkjaoigfuey** [comment]
确实是没事做

**Comment by AnonCN** [comment]
我不仅长草，我青衣还歪了（悲）

**Reply by LT-UT** [reply]
祝你下次小保底不歪
<img src="https://img-static.mihoyo.com/communityweb/upload/00621d7e9026af7568e8382def4d2dae.gif" alt="米游姬 开心" width="75" >

**Comment by cen-zzi** [comment]
可以尝试使用你喜欢的角色去单通（
最近才用朱鸢单通了尼尼微烈度11，新版本去凹新深渊八单通了

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
买点百草枯撒上，草就没了（doge）
